libr/core/panels.c

```diff
@@ -929,7 +931,8 @@ R_API int r_core_visual_panels(RCore *core) {
 	case 'K':
 		menu_y = 0;
 		menu_x = -1;
-		curnode--;
+		panels[curnode--].refresh = true;
```

Comment:
While technically is correct, I think the point of decrementing curnode can be missed while reading such code. I suggest you to split it to the two lines:
```c
panels[curnode].refresh = true;
curnode--;
```
You may argue that experienced C developer will catch this instantly, but lets think about also inexperienced C developers, experienced but drunk or very tired C developers, etc.

Reply:
lmao yeah there must be bunch of them workdwide let me quickly fix that up